### PR TITLE
github practice

### PR DESCRIPTION
**Pull request by AniketNeogy**
github practice

### CODE REVIEW

File: src/Day4/TwoLinkedListIntersection.java [hunk 1]
```diff
@@ -49,19 +49,13 @@ public ListNode getIntersectionNode(ListNode headA, ListNode headB) {
...
 		//Linked List 1
-		ListNode  anewNode6 = new ListNode(5, null);
-		ListNode  anewNode5 = new ListNode(4, anewNode6);
-		ListNode  anewNode4 = new ListNode(8, anewNode5);
-		ListNode  anewNode3 = new ListNode(1,anewNode4);
-		ListNode  anewNode2 = new ListNode(6,anewNode3);
-		ListNode  head1 = new ListNode(5,anewNode2);
+		ListNode  anewNode3 = new ListNode(6,null);
+		ListNode  anewNode2 = new ListNode(4,anewNode3);
+		ListNode  head1 = new ListNode(2,anewNode2);
```
Comment (AniketNeogy) [inline src/Day4/TwoLinkedListIntersection.java]: Modified the input list size

File: src/Day4/TwoLinkedListIntersection.java [hunk 2]
```diff
@@ -49,19 +49,13 @@ public ListNode getIntersectionNode(ListNode headA, ListNode headB) {
...
 
 		//Linked List 2
-		ListNode  bnewNode5 = anewNode6;
-		ListNode  bnewNode4 = anewNode5;
-		ListNode  bnewNode3 = anewNode4;
-		ListNode  bnewNode2 = new ListNode(1,bnewNode3);
-		ListNode  head2 = new ListNode(4,bnewNode2);
+		ListNode  bnewNode2 = new ListNode(5,null);
+		ListNode  head2 = new ListNode(1,bnewNode2);
```
Comment (AniketNeogy) [inline src/Day4/TwoLinkedListIntersection.java:59]: Modified the list items

File: src/Day4/TwoLinkedListIntersection.java (hunk 1 above)
Comment (shaliniKar3) [inline src/Day4/TwoLinkedListIntersection.java]: Best to give larger inputs

File: src/Day6/BackspaceStringCompare.java [hunk 3]
```diff
@@ -39,9 +39,13 @@
 
 package Day6;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Stack;
 
 public class BackspaceStringCompare {
+	
+	List<Integer> ls = new ArrayList<Integer>();
```
Comment (shaliniKar3) [inline src/Day6/BackspaceStringCompare.java:48]: Can implement this using java linkedlist class

**Review by shaliniKar3 — CHANGES_REQUESTED** [review]

**Review by shaliniKar3 — APPROVED** [review]

**Pull request merged**